### ISSUE TITLE
Preserve extension

### ISSUE THREAD

**Issue by normalser** [mrmlnc/vscode-duplicate#1]
@mrmlnc - Thanks so much.
This is awesome - and the fact that I can put path when renaming is brilliant - wish vscode had that rename with path ability as well.
One small request if possible - in 99% cases I guess people will want to keep the original extension so maybe the plugin could do the same what Atom does (extension is part of the file - but just name is highlighted)
![image](https://cloud.githubusercontent.com/assets/1723815/17078037/ebc9a426-50b5-11e6-8698-89941c98595c.png)

**Comment by mrmlnc** [comment]
@wallverb , you can save the file extension, simply not adding new extension.

![2016-07-23_16-22-42](https://cloud.githubusercontent.com/assets/7034281/17078093/ba4cfb1e-50f1-11e6-8658-54cb4ee6f6d7.gif)

Unfortunately, i couldn't find a way to make a selection in the input form. Later i'll try to come back to search again.

**Comment by normalser** [comment]
That's a fantastic hint - works perfectly. Thanks so much.
Maybe would be worth to put that hint into doc - I read it - but for some reason I assumed that it will create copy of the file without extension (although I should have tried it first).

**Comment by mrmlnc** [comment]
I'll be very grateful to you for PR. :smile_cat:

**Comment by mrmlnc** [comment]
Fixed in version 1.0.1.

# 

If you use this extension and satisfied with his work, then please leave a review in the marketplace.

**Comment by druellan** [comment]
Hi!
Version 1.0.2, I'm duplicating double-extension filenames like `contentblock-type-brand-assets.inc.php` and I'm having trouble preserving the ".php".

Hitting "enter" at:

![image](https://cloud.githubusercontent.com/assets/6965667/21868391/4e923390-d831-11e6-81fd-055910cbaac9.png)

The extension is not preserved:

![image](https://cloud.githubusercontent.com/assets/6965667/21868406/61762282-d831-11e6-80c3-852e1d63563f.png)
